examples/boolean_project_allow/main.tf

```diff
@@ -18,7 +18,7 @@
   Provider configuration
  *****************************************/
 provider "google" {
-  credentials = "${file(var.credentials_file_path)}"
+  credentials = file(var.credentials_file_path)
```

Comment:
along with linter should it be 
```
provider "google" {
}
```
or 
`provider "google" {}`
?

Reply:
sorry, I didn't get that, can you be more specific?